tests/framework/jailer.py

```diff
@@ -75,7 +75,7 @@ def __del__(self):
         self.cleanup()
 
     # Disabling 'too-many-branches' warning for this function as it needs to
-    # check every argument, so the number of branches will be increase
+    # check every argument, so the number of branches will increase.
```

Comment:
I think this full-stop is not needed. Sorry if it was a suggestion based on my full-stops paranoia.

Reply:
Ok, no problem, I'll remove it.